### ISSUE TITLE
Question: Rollback last finished migration ?

### ISSUE THREAD

**Issue by mablo** [cakephp/phinx#1355]
Some pr are merged in a different order than the migration id. This causes situations where there is no possibility to rollback last finished migration.

Example:
Status:
```
 Status  [Migration ID]  Started              Finished             Migration Name
----------------------------------------------------------------------------------
     up  20170914120527  2017-11-07 10:40:20  2017-11-07 10:40:20  Init
     up  20180420080835  2018-05-08 13:45:37  2018-05-08 13:45:38  A
     up  20180510082825  2018-05-24 11:11:23  2018-05-24 11:11:25  B
     up  20180510100510  2018-05-23 16:02:55  2018-05-23 16:02:55  C
```
Migrations: Init -> A -> C -> B
How to rollback only "B"  ?

**Comment by rquadling** [comment]
@mablo Did that make sense? I accidentally closed the issue before waiting on feedback.

**Comment by othercorey** [comment]
Closing due to lack of activity or out of scope. If there is new interest, please open a PR.
